data_store/const.py

```diff
@@ -746,3 +746,19 @@ class FundTypeIdEnum(StrEnum):
     "Northampton": {"Town_Deal", "Future_High_Street_Fund"},
     "Barrow": {"Town_Deal"},
 }
+
+# Maps Pathfinder reporting round numbers to observation window start and end dates
+PF_REPORTING_ROUND_NUMBER_TO_OBSERVATION_DATES = {
+    1: {
+        "start": datetime(2024, 1, 1),
+        "end": datetime(2024, 3, 31, 23, 59, 59),
+    },
+}
+
+# Maps Pathfinder reporting round numbers to submission period start and end dates
+PF_REPORTING_ROUND_NUMBER_TO_SUBMISSION_DATES = {
+    1: {
+        "start": None,
+        "end": datetime(2024, 4, 30, 23, 59, 59),
+    },
+}
```

Comment:
is this only required until the reportinground changes I'm working on go in? 👀

Reply:
Aye